docs/decorators.md

```diff
@@ -635,6 +635,6 @@ ImportError: cannot import name 'get_val' from 'boo' (/home/__init__.py)
 
 ## Low-level
 
-This decorator is not designed to be used in a `.py` file. Rather it should be used in `.pyi` files. These files are generated by Pyccel to speed up multi-file compilation. They can also be used with `pyccel-wrap` to expose existing Fortran or C code to Python. See [Header Files](./header-files.md) for more information.
+This decorator is not designed to be used in a `.py` file. Rather it should be used in `.pyi` files. These files are generated by Pyccel to speed up multi-file compilation. They can also be used with `pyccel-wrap` to expose existing Fortran or C code to Python. See [Interfacing Python code with low-level code](./low-level-interface.md) for more information.
```

Comment:
In this paragraph we do not say what this decorator means. I suggest we explain it in a few words.

Reply:
Fixed [3249ca6](https://github.com/pyccel/pyccel/pull/2422/commits/3249ca622f101a9b1b43b3c78261b1bfefc5bf04)